CampaignConfig/mdc2020_cry.cfg

```diff
@@ -1,9 +1,9 @@
 [global]
-includes = /mu2e/app/users/sophie/ProductionPush/Production/CampaignConfig/mdc2020_prolog.cfg
-desc = MDC2020v
+includes = /cvmfs/mu2e.opensciencegrid.org/Musings/SimJob/MDC2020x/Production/CampaignConfig/mdc2020_prolog.cfg
+desc = MDC2020x
```

Comment:
Is it time to look ahead and say that this will be MDC2020y ?

Until know I had not appreciated how many places haded to be fixed by hand to make this work.  I knew it was a lot but I had not seen i so clearly until now.   Does it make sense to bring this to the POMS team and ask they can see new POMS feature that would simplify this use case?

Reply:
yes it would be great is there were just "version" parameter in the prolog or something and that we could have a %(version)s in the paths, but I dont think that works in the CFG paths

Reply:
and yes, I guess this will need to be MDC2020y, once I have all the pieces, I'll change it